### DISCUSSION TITLE
Allow more browsers to log in

### DISCUSSION THREAD

**Discussion by dickwolff** [f1multiviewer/issue-tracker#133] [Feature requests] [answered]
Currently there are two options to log in, via built in browser to F1TV login or to use Chrome.
![Login view](https://user-images.githubusercontent.com/5620002/181754533-66ac51df-12d7-4950-b956-bac64e3adb8f.png)

For users like me who don't use Chrome (I don't even have it installed) it would be nice to have a login button that supports my browsers. As I use Brave Browser, which is based on Chrome, it would be nice to have a button to log in using Brave. 

To have a more user friendly option, it would be nice to have a "Sign in using default browser" option, so no matter which browser you have it will just open in your favorite browser.

**Comment by jeppeolesen** [comment, marked as answer]
There's now a companion browser extension for all major browsers that allow you to sign in that way: https://multiviewer.app/extension
